src/main/java/org/killbill/billing/plugin/adyen/api/mapping/KlarnaPaymentMappingService.java

```diff
@@ -65,6 +77,46 @@ public static PaymentInfo toPaymentInfo(final String merchantAccount, final Stri
         return lineItems;
     }
 
+    private static List<LineItem> getLineAdjustments(final List<LineItem> lineItems) {
+        final Map<String, LineAdjustment> adjustmentMap = new HashMap<>();
+        final List<LineItem> adjustmentLines = new ArrayList<>();
+        if(lineItems!= null && lineItems.size() > 0) {
+            for(LineItem lineItem: lineItems) {
+                List<LineAdjustment> adjustments = lineItem.getAdjustments();
+                if(adjustments != null && adjustments.size() > 0) {
```

Comment:
I guess adjustments != null check is enough

Reply:
+1